testing/py-virtualenvwrapper/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Herve Beraud <herveberaud.pro@gmail.com>
+# Maintainer: Herve Beraud <herveberaud.pro@gmail.com>
+pkgname=py-virtualenvwrapper
+pkgver=4.8.0
+pkgrel=0
+pkgdesc="Enhancements to virtualenv"
+url="https://bitbucket.org/virtualenvwrapper/virtualenvwrapper/"
+arch="noarch"
+license="non-free"
+depends="python"
+makedepends='python2-dev python3-dev py-setuptools py-pbr py2-pbr'
+install=""
+#subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
+subpackages=""
+source="${pkgname}-${pkgver}.tar.gz::https://bitbucket.org/virtualenvwrapper/virtualenvwrapper/get/$pkgver.tar.gz"
+builddir="$srcdir/"
+
+build() {
+	msg "Building python"
+	cd "$builddir/virtualenvwrapper-virtualenvwrapper-c9300524289e"
```

Comment:
`builddir=...` is there to make future updates easier. Now for every update both sections must be modified.